actionpack/lib/action_controller/metal/request_forgery_protection.rb

```diff
@@ -262,7 +262,7 @@ def verified_request?
       end
 
       # Sets the token value for the current session.
-      def form_authenticity_token
+      def form_authenticity_token(action_path=nil, method=nil)
```

Comment:
Passing the form html options as keyword args would make it easy for others to do custom tokens and give some measure of forward-compat.